### PR TITLE
Fix CA parser to parse gas values 

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Issue

image

due to a casing issue, we were not getting gas production for 2020 and most of 2021
image

## Description
 

the columns of the df parsed in fetch_production are set to lowercase to prevent any casing error in the future. 
the parser is then fit to the parser validation scheme

### Preview

image

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Review by VIKTORVAV99 — COMMENTED** [review]
Seems to work well, so of you fix the types and remove the unused imports I think this can be merged.

File: parsers/US_CA.py [hunk 1]
```diff
@@ -257,12 +241,12 @@ def fetch_exchange(
     from pprint import pprint
 
     print("fetch_production() ->")
-    pprint(fetch_production())
+    pprint(fetch_production(target_datetime=datetime(2020, 1, 20)))
 
-    print('fetch_exchange("US-CA", "US") ->')
-    # pprint(fetch_exchange("US-CA", "US"))
+    # print('fetch_exchange("US-CA", "US") ->')
```
Comment (unitrium) [inline parsers/US_CA.py:246]: I would recommend not changing this part.

**Review by unitrium — COMMENTED** [review]
Looks good to me! Feel free to ping me once you have been able to implement @VIKTORVAV99 suggestions.

**Review by unitrium — APPROVED** [review]
Looks good!

**Comment by VIKTORVAV99** [comment]
Please check the lint issue before merging 🙂

**Pull request merged**